### PR TITLE
Design doc edits

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
The intro paragraph for buttons appears twice.  

Everything else looks good to me, but I'm assigning to @Blackbaud-ToddRoberts, since he, @Blackbaud-AdamFunderburk, and @Blackbaud-MattGarbarino provided much of the content.

File: content/core-design/typography/index.html [hunk 1]
```diff
@@ -1,63 +1,63 @@
...
 
   - name: Headline
-    usage: This is used to call out key always-relevant pieces of information on page, such as a constituent’s total giving or the number of records in a list. It should be used sparingly or else the attention-drawing aspect will be diluted.
+    usage: This class formats text that calls out key information on a page, such as a constituent’s total giving or the number of records in a list. Use it sparingly or you will dilute its ability to draw attention.
     code: <div class="bb-headline">Headline</div>
     styling: Oswald Regular 20px #292a2b
 
   - name: Emphasized body copy
-    usage: This is used to call out pieces of information that are important but not always what the user is looking for, such as a constituent’s name in a list of contact cards. It should be used sparingly or else the attention-drawing aspect will be diluted.
+    usage: This class formats text call out key information that is slightly less important than headline text, such as constituent names in a list of contact cards. Use it sparingly or you will dilute its ability to draw attention.
```
Comment (Blackbaud-ToddRoberts) [inline content/core-design/typography/index.html]: missing a word between "text" and "call"

File: content/core-design/typography/index.html [hunk 2]
```diff
@@ -1,63 +1,63 @@
...
     code: <div class="bb-emphasized">Emphasized body copy</div>
     styling: Open Sans Semibold 14px #292a2b
 
   - name: Regular body copy
-    usage: The default font, used when the data being presented does not semantically fall under one of the defined types.
+    usage: This the base body font and does not require a separate class. It formats text that does not semantically fall under any of the defined types.
```
Comment (Blackbaud-ToddRoberts) [inline content/core-design/typography/index.html]: missing an "is"

**Comment by Blackbaud-ToddRoberts** [comment]
Fixed a couple typos.

The "Calling out information" guideline was intentionally written to be scenario-oriented rather than component-oriented, the proposed changes go against that.

**Pull request merged**